### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by DanielMeixner**
Updated debugger.md to latest instructions and did some minor adjustments.
Added debugger section to readme and a pointer to the "old" extension.

### CODE REVIEW

**Comment by gregg-miskelly** [comment]
This PR has conflicts and should really have a better name. Let me know when you are back to your office and I can help you fix these.

**Comment by gregg-miskelly** [comment]
@caslan please review also

File: debugger.md [hunk 1]
```diff
@@ -26,17 +26,17 @@ Please place your feedback [here](https://github.com/OmniSharp/omnisharp-vscode/
 2. Install Dotnet CLI
  * Install Dotnet CLI following the instructions here:  http://dotnet.github.io/getting-started  
  * **Hint for Mac**: Dotnet CLI requires openSSL to work. It will ask you to install this during installation. Don't forget this!
-3. C# Extension for VS Code
- * Get the .vsix file from here: \\\\vsdbgqa\x-plat\vscode-extension\csharp
+3. Install C# Extension for VS Code
+ * Get the .vsix file from here: \\\\vsdbgqa\x-plat\vscode-extension\osx\csharp\csharp-0.3.5.vsix .
```
Comment (caslan) [inline debugger.md]: Let's put the osx one under \\vsdbgqa\x-plat\vscode-extension and not bother telling people of osx in the path etc.

File: debugger.md [hunk 2]
```diff
@@ -45,12 +45,11 @@ The following steps have to executed for every project.
...
-  * ~~Or you can open the command pallette in VSC (F1) and run the command *"Debugger: Add tasks.json"* (Make sure you have installed Mono to be able to do so, see above.
-  (If you don't have Mono installed you might get an error saying *Omnisharp not running*)~~ Unfortunately this is currently not working. We're investigating why.
+  * Or you can open the command pallette in VSC (F1) and run the command *"Debugger: Add tasks.json"* (Make sure you have installed Mono to be able to do so, see above. If you don't have Mono installed you might get an error saying *Omnisharp not running*.)
   * Or if you just want debugging, you can comment out 'preCompileTask' field​
-* **Important hint:**If you run into an error modify your project.json to reference "NETStandrad.Library":"1.0.0-rc3-*"
-  * Make sure the version number is correct. Currently dotnet new creates a project.json that references a version that's too old. You should also get a hint to modify this.
-  * Afterwards run dotnet restore.
+  * Modify your project.json to reference "NETStandrad.Library":"1.0.0-rc3-*"
```
Comment (caslan) [inline debugger.md]: typo: NETStandrad

**Comment by gregg-miskelly** [comment]
@Andrew-MSFT @richlander do you want to look at the readme part?

File: README.md [hunk 3]
```diff
@@ -1,7 +1,8 @@
 ## C# for Visual Studio Code (powered by OmniSharp)
 
-Don't install this extension unless you want to try an early version. By default it is already
-bundled with Visual Studio Code.
+Don't install this extension unless you want to try an early version.
```
Comment (gregg-miskelly) [inline README.md]: How about something like --
This is a preview of C# support in Visual Studio code. It is designed to work well with upcoming .NET Core 1.0 -- see link for more information on what .NET Core 1.0 is. These new projects are based on project.json and the .NET CLI (links).

For msbuild based projects, or ASP.NET Core 5.0 you still want to use the omnisharp extension instead (link).

File: debugger.md [hunk 4]
```diff
@@ -15,8 +15,8 @@ Please place your feedback [here](https://github.com/OmniSharp/omnisharp-vscode/
 * X64 only
 * Basic Debugging Setup successfully validated on Ubuntu Linux and Mac
 * Currently NOT working on Windows
-* In case you get this error on F5: *"No task runner configured -  Tasks.json could not be found"*, see the 'once for each project' section below. Currently manual steps are required.
-* If you don’t have mono installed you won’t get IntelliSense. See optional installation section below.
+* In case you get this error on F5: *"No task runner configured -  Tasks.json could not be found"*, see the 'once for each project' section below. 
+* If you don’t have mono installed you won’t get IntelliSense.
```
Comment (gregg-miskelly) [inline debugger.md]: you have a 'word quote'. Might want to convert these to normal (I don't know how well they will render on other OSs).

File: README.md [hunk 5]
```diff
@@ -1,7 +1,8 @@
 ## C# for Visual Studio Code (powered by OmniSharp)
 
-Don't install this extension unless you want to try an early version. By default it is already
-bundled with Visual Studio Code.
+Don't install this extension unless you want to try an early version. 
+
```
Comment (gregg-miskelly) [inline README.md]: Should we add some screen captures like https://github.com/microsoft/vscode-go?

File: README.md [hunk 6]
```diff
@@ -12,6 +13,10 @@ To **run and develop** do the following:
 * *Optional:* run `tsc -w`, make code changes (on Windows, try `start node ".\node_modules\typescript\bin\tsc -w"`)
 * Press F5 to debug
 
+### Debugging
```
Comment (gregg-miskelly) [inline README.md]: 1. This should go above 'Development'
2. As I think about this more, I wonder if we want to actually pull most of the content into this file instead - a lot of the content is really about installation which is common to any usage of the extension

File: README.md [hunk 7]
```diff
@@ -1,10 +1,17 @@
 ## C# for Visual Studio Code (powered by OmniSharp)
 
-Don't install this extension unless you want to try an early version. By default it is already
-bundled with Visual Studio Code.
+Don't install this extension unless you want to try an early version. 
 
-### Development
+This is a preview of C# support in Visual Studio code. It is designed to work well with upcoming [.NET Core 1.0]([link](http://dotnet.github.io/).
```
Comment (gregg-miskelly) [inline README.md]: I think you want to remove the '[link]' text after '[.NET Core 1.0]'. I am not sure why it works.

**Comment by gregg-miskelly** [comment]
@DustinCampbell do you want to review the readme.md part of this? Or are you okay if I hit merge?

**Comment by DustinCampbell** [comment]
LGTM

**Pull request merged**